Formula/fastfetch.rb

```diff
@@ -17,10 +17,14 @@ class Fastfetch < Formula
     sha256 x86_64_linux:   "8757d24b5daa28af495feec81f4be8fa5952c23df8f24185d8a0c69b5dc9269e"
   end
 
+  depends_on "chafa" => :build
   depends_on "cmake" => :build
+  depends_on "imagemagick" => :build
   depends_on "pkg-config" => :build
   depends_on "vulkan-loader" => :build
 
+  uses_from_macos :zlib
```

Comment:
This shouldn't be a symbol
```suggestion
  uses_from_macos "zlib"
```